bls_test.go

```diff
@@ -121,12 +121,10 @@ func BenchmarkBLSVerifyBatch(b *testing.B) {
 func benchmarkBLSVerifyBatchSize(size int) func(b *testing.B) {
 	return func(b *testing.B) {
 		var digests []Digest
-		var msgs []Message
 		var sigs []Signature
 		var pubks []PublicKey
 		for i := 0; i < size; i++ {
 			msg := Message(fmt.Sprintf("cats cats cats cats %d %d %d dogs", i, i, i))
-			msgs = append(msgs, msg)
```

Comment:
Is this expected? I guess that breaks some linting rules?

Reply:
Yeah, the linting rules started failing. The issue made sense to me so I fixed it. I suspect we're using newer version of some linting tool or something like that.

Reply:
https://hub.docker.com/layers/cimg/go/1.21.2-node/images/sha256-59b3fff1c4745e9195ea35eef2cfbee4cb0b562616869e304b581644020ad4b8 - golangci-lint v1.54.2 vs v1.56.2

but, I don't see any version of golangci-lint failing on this file. staticcheck does but I don't see it being used here?

Reply:
btw I'm not objecting to this fix, just interested in why this is showing up for this diff

Reply:
This was the failing run before I added this change - https://github.com/filecoin-project/filecoin-ffi/actions/runs/9805936223/job/27076653921

It does come from `staticcheck` as you pointed out. I wonder why we weren't running it in CircleCI.

Reply:
It is listed in the linters list there, too. Honestly, I'm not sure why it wasn't flagging this issue.

Reply:
`make go-lint` only runs golangci-lint. So how about we add staticcheck install to this workflow and to the Makefile too so we have it properly covered

Reply:
It is running in GHA - that's why I had to add the fix - so I wonder how much time we want to spend investigating this.